src/usr/local/www/services_dyndns_edit.php

```diff
@@ -491,6 +492,9 @@ function setVisible(service) {
 				hideInput('ttl', false);
 				break;
 			case "namecheap":
+			case "cloudflare-v6":
+			case "cloudflare":
```

Comment:
The block just under this also has cloudflare inside:

https://github.com/pfsense/pfsense/blob/master/src/usr/local/www/services_dyndns_edit.php#L472

https://github.com/pfsense/pfsense/blob/master/src/usr/local/www/services_dyndns_edit.php#L489